### PR TITLE
Custom rollup trace

### PR DESCRIPTION
**Pull request by VinnyQ**
add loggings to debug rollup

### CODE REVIEW

**Comment by usnavi** [comment]
fyi - locators per shard tracking was commited to master in https://github.com/rackerlabs/blueflood/pull/733  

Might be better to rebase/merge the conflicts.

File: blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/LocatorFetchRunnable.java [hunk 1]
```diff
@@ -101,7 +103,7 @@ public void run() {
         final RollupBatchWriter rollupBatchWriter = createRollupBatchWriter(executionContext);
 
         Set<Locator> locators = getLocators(executionContext);
-
+        log.debug("locators retrieved: {}", locators.size());
```
Comment (ChandraAddala) [inline blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/LocatorFetchRunnable.java]: Can you set this to trace level?

File: blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/LocatorFetchRunnable.java [hunk 2]
```diff
@@ -189,7 +191,8 @@ public void executeRollupForLocator(RollupExecutionContext executionContext, Rol
         try {
             // get a list of all locators to rollup for a shard
             locators.addAll(IOContainer.fromConfig().getLocatorIO().getLocators(getShard()));
-        } catch (Exception e) {
+            locatorsPerShard.update(locators.size());
```
Comment (ChandraAddala) [inline blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/LocatorFetchRunnable.java]: Can you add `log.error` to this `catch` statement too?

File: blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/astyanax/ALocatorIO.java [hunk 3]
```diff
@@ -72,13 +72,14 @@ public void insertLocator(Locator locator) throws IOException {
             RowQuery<Long, Locator> query = AstyanaxIO.getKeyspace()
                     .prepareQuery(CassandraModel.CF_METRICS_LOCATOR)
                     .getKey(shard);
+            LOG.debug("ALocatorIO.getLocators() executing: select * from \"" + CassandraModel.KEYSPACE + "\"." + CassandraModel.CF_METRICS_LOCATOR + " where key=" + Long.toString(shard));
```
Comment (ChandraAddala) [inline blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/astyanax/ALocatorIO.java]: Can you set this to trace?

File: blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/LocatorFetchRunnable.java (hunk 2 above)
Comment (VinnyQ) [inline blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/LocatorFetchRunnable.java]: it's already there but below the executionContext.markUnsuccessful(e); call.  Let me move it up.

File: blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/LocatorFetchRunnable.java (hunk 1 above)
Comment (VinnyQ) [inline blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/LocatorFetchRunnable.java]: done.

File: blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/astyanax/ALocatorIO.java (hunk 3 above)
Comment (VinnyQ) [inline blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/astyanax/ALocatorIO.java]: done.

**Comment by shintasmith** [comment]
lgtm 👍

**Comment by shintasmith** [comment]
merging in for @VinnyQ

**Pull request merged**